CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ Please add to the relevant subsections under Unreleased below on every PR where
 * Implemented a stricter check for AMD GPU availability by @dmitryduev in https://github.com/wandb/wandb/pull/7322
```

Comment:
should we document the settings that you removed?

Reply:
good point, done